landlab/components/landslides/landslide_probability.py

```diff
@@ -366,14 +404,14 @@ def __init__(
             'lognormal', 'lognormal_spatial,' or 'data_driven_spatial'.
             (default='uniform')
         groundwater__recharge_min_value: float, optional (mm/d)
```

Comment:
Another thought would be that rather than duplicating all of the options for groundwater depth and groundwater recharge you could have:

variable (recharge or depth)
distribution (uniform, lognormal, lognormal_spatial, or data_driven_spatial)
min_val (for uniform)
max_val (for uniform)
mean_val (for lognormal and lognormal spatial)
std_val for lognormal and lognormal spatial)
HSD inputs (for data driven spatial)


This would help reduce overspecification (discussed more below)